api/src/models/massageLaw.js

```diff
@@ -0,0 +1,13 @@
+/** Schema representing a massage law **/
+const mongoose = require('mongoose');
+
+const MassageLaw = new mongoose.Schema({
+  state: { type: String, required: false },
+  city: { type: String, required: false },
+  strengthOfStateLaw: { type: String, required: false },
```

Comment:
nit: it could technically be a city law, so you can probably change to just "strengthOfLaw"